openrr-apps/Cargo.toml

```diff
@@ -30,3 +32,7 @@ rand = {version="0.8.0", optional = true}
 [[bin]]
 name = "openrr_apps_robot_command"
 path = "src/bin/robot_command.rs"
+
+[[bin]]
+name = "openrr_apps_robot_teleop"
```

Comment:
Comment:
openrr contains robot, so i think we can remove robot from the name.
(But keep this for consistency with robot_command for now)